src/update_engine/delta_diff_generator.cc

```diff
@@ -1243,6 +1353,8 @@ bool DeltaDiffGenerator::GenerateDeltaUpdateFile(
 
   LOG(INFO) << "Reading files...";
 
+  vector<InstallOperation> kernel_ops;
```

Comment:
How is this ever longer than size 1 (after DeltaCompressKernel)? Will it ever be?